data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Migrate MySQL Data of More than 1 TiB to TiDB
```

Comment:
@sunzhaoyang What do you think? Is `TB 级以上` a general or specific description?
Note that we need to keep the description consistent across other data migration docs as well, e.g. https://github.com/pingcap/docs/pull/7202/files

Reply:
I think we just want to express user should use different approach for different scale of source data:
- Large scale. dumpling+lightning -> dm (incremetal). This apporch is fast, but need more operations.
- Small scale. dm (batch + incremental). This approach is much slower, but easier to use.

IMHO, `1 TB`  is not a very good threshold here because I think we won't suggest user to only depend dm to migrate 999GB source data bacause this will cost at least 1day to finish migration. 
Users themselves will determine whether their data scale is large or small, either based on intuition or  the benchmark result.

No sure if we can use other word to deliver a more precise and understandable title.